src/Fixer/CastNotation/CastSpacesFixer.php

```diff
@@ -56,22 +71,17 @@ public function isCandidate(Tokens $tokens)
      */
     protected function applyFix(\SplFileInfo $file, Tokens $tokens)
     {
-        static $insideCastSpaceReplaceMap = [
-            ' ' => '',
-            "\t" => '',
-            "\n" => '',
-            "\r" => '',
-            "\0" => '',
-            "\x0B" => '',
-        ];
-
         foreach ($tokens as $index => $token) {
-            if ($token->isCast()) {
-                $tokens[$index] = new Token([
-                    $token->getId(),
-                    strtr($token->getContent(), $insideCastSpaceReplaceMap),
-                ]);
+            if (!$token->isCast()) {
+                continue;
+            }
+
+            $tokens[$index] = new Token([
+                $token->getId(),
+                strtr($token->getContent(), self::INSIDE_CAST_SPACE_REPLACE_MAP),
+            ]);
 
+            if ($this->configuration['space'] === 'single') {
```

Comment:
please make this Yoda as we follow this rule in the repo, after that I'm :+1: for merging this :)
thanks for picking this up